administrator/language/de-DE/joomla.ini

```
@@ -1027,7 +1027,7 @@ JWARNING_UNPUBLISH_MUST_SELECT="Es muss mindestens ein Beitrag zum Verstecken au
 JWORKFLOW="Workflow: %s"
 JWORKFLOW_ENABLED_LABEL="Workflow aktivieren"
 JWORKFLOW_EXECUTE_TRANSITION="Übergang für dieses Element auswählen"
-JWORKFLOW_EXTENSION_FORBIDDEN_DESCRIPTION="Dieses Plugin für aufgelistete Erweiterungen deaktivieren."
+JWORKFLOW_EXTENSION_FORBIDDEN_DESCRIPTION="Dieses Plugin für aufgelistete Erweiterungen deaktivieren"
```

Siehe Core: https://github.com/joomla/joomla-cms/blob/184cc04809747470578de78cb448392a63da2eb0/administrator/language/en-GB/joomla.ini#L1028

```suggestion
JWORKFLOW_EXTENSION_FORBIDDEN_DESCRIPTION="Dieses Plugin für aufgelistete Erweiterungen deaktivieren."
```